.github/workflows/rust.yml

```diff
@@ -83,7 +83,7 @@ jobs:
       - name: Run sccache-cache
         uses: mozilla-actions/sccache-action@v0.0.9
       - name: Run tests for main workspace
-        run: cargo test
+        run: cargo test --all-targets --features integration_test
```

Comment:
perhaps worth breaking out integration_test into a separate job if we plan to build out more integration tests?